server_addon/aftereffects/client/ayon_aftereffects/plugins/publish/collect_render.py

```diff
@@ -173,24 +181,25 @@ def get_expected_files(self, render_instance):
             _, ext = os.path.splitext(os.path.basename(file_name))
             ext = ext.replace('.', '')
             version_str = "v{:03d}".format(render_instance.version)
-            if "#" not in file_name:  # single frame (mov)W
-                path = os.path.join(base_dir, "{}_{}_{}.{}".format(
-                    render_instance.folderPath,
+            if "#" not in file_name:  # single frame (mov)
+                file_name = "{}_{}.{}".format(
                     render_instance.productName,
                     version_str,
                     ext
-                ))
-                expected_files.append(path)
+                )
+                file_path = os.path.join(base_dir, file_name)
+                expected_files.append(file_path)
             else:
                 for frame in range(start, end + 1):
-                    path = os.path.join(base_dir, "{}_{}_{}.{}.{}".format(
-                        render_instance.folderPath,
+                    file_name = "{}_{}.{}.{}".format(
                         render_instance.productName,
                         version_str,
                         str(frame).zfill(self.padding_width),
                         ext
-                    ))
-                    expected_files.append(path)
+                    )
+
+                file_path = os.path.join(base_dir, file_name)
+                expected_files.append(file_path)
```

Comment:
I guess this should be indented one more to add all frames to `expected_files`.
```suggestion
                    file_path = os.path.join(base_dir, file_name)
                    expected_files.append(file_path)
```

Reply:
Shhh...fixed